test/__init__.py

```diff
@@ -919,6 +924,32 @@ def _target() -> None:
             self.assertEqual(proc.exitcode, 0)
 
 
+class UnitTest(PyMongoTestCase):
+    """Async base class for TestCases that don't require a connection to MongoDB."""
+
+    @classmethod
+    def setUpClass(cls):
+        if _IS_SYNC:
```

Comment:
Same comment. How does/may _IS_SYNC get redefined when tests are running?

Reply:
`_IS_SYNC` is always true for a synchronous file and always false for an asynchronous file.